benchmark/events/ee-emit.js

```diff
@@ -3,20 +3,27 @@ const common = require('../common.js');
 const EventEmitter = require('events').EventEmitter;
 
 const bench = common.createBenchmark(main, {
-  n: [2e6],
+  n: [2e7],
   argc: [0, 2, 4, 10],
   listeners: [1, 5, 10],
 });
 
 function main(conf) {
-  const n = conf.n | 0;
+  var n = conf.n | 0;
   const argc = conf.argc | 0;
   const listeners = Math.max(conf.listeners | 0, 1);
 
   const ee = new EventEmitter();
 
-  for (var k = 0; k < listeners; k += 1)
+  if (listeners === 1)
+    n *= 5;
```

Comment:
I don't think automagically adjusting the `n` value is a good idea.

Reply:
I mean, we don't have a ton of options here. The benchmark doesn't run long enough with 1 listener whereas setting a higher default `n` makes the other ones run way too long.

Reply:
That's just a benchmarking problem in general, just like any particular `n` value might cause things to run too fast or too slow on someone else's machine. At some point you will just need to supply your own set of values for each parameter if you have combinations that vary in speed that much.